ckanext/recombinant/tests/test_tables.py

```diff
@@ -0,0 +1,167 @@
+import os
+import pytest
+
+from ckan.tests.helpers import change_config
+from ckan.plugins.toolkit import config
+
+from ckanext.recombinant.tests import RecombinantTestBase
+from ckanext.recombinant.tables import (
+    get_dataset_types,
+    get_geno,
+    get_chromo,
+    get_resource_names,
+    get_published_resource_resource_name,
+    get_dataset_type_for_resource_name,
+    get_target_datasets,
+    _get_plugin
+)
+from ckanext.recombinant.errors import RecombinantException
+
+
+class TestRecombinantTables(RecombinantTestBase):
+    @classmethod
+    def setup_method(self, method):
+        """Method is called at class level before EACH test methods of the class are called.
+        Setup any state specific to the execution of the given class methods.
+        """
+        super(TestRecombinantTables, self).setup_method(method)
+
+        self.expected_chromo = {
```

Comment:
this could be a class variable instead of a `setup_method` unless you're modifying it in the tests

Reply:
Nope, not modifying in the test methods. Moving to class level